### PR TITLE
Submitting HW1

### PR DESCRIPTION
**Pull request by ezradmax**
Hi Professor,

I hope I've done everything right and that you can read my files etc. If you have a moment to check, could you verify that I've submitted everything correctly? I am new to all this and want to make sure I didn't misinterpret your instructions for either formatting my work or submitting it. If you don't have time to do this, no worries.

Thanks,
Ezra Max

### CODE REVIEW

**Comment by dhruvalb** [comment]
Evaluation
----------------------------------------------------
| Topic                       | Total | Score| Comments |
|-----------------------------|-----------|--------------|------------|
| **Statistical and Machine Learning** |   25        |      25        |            |
| **Linear Regression (a)** |   10        |     10         |            |
| **Linear Regression (b)** |   5        |        5      |     equation shown in part a.        |
| **Linear Regression (c)** |   10        |      10        |            |
| **Linear Regression (d)** |   10        |      10        |            |
| **Non-Linear Regression (a)**            |   10 |      10        |   good use of train/test split. (answers were slightly different from the key as the seed to split is different but right method and relation.)         | 
| **Non-Linear Regression (b)**            |   10        |     10         |            |
| **Non-Linear Regression (c)**            |   10        |     10         |            | 
| **Non-Linear Regression (d)**            |   10        |     10         |            |  
| **Total**            |   100        |     100         |     Good work showing all the work and using mean squared error for model evaluation.        |  

Remarks: 
For the train, test split in pandas this is an easy way: 
from sklearn.model_selection import train_test_split 
X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=0.2, random_state=0)
